package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@types/validator",
+  "version": "0.0.1",
```

Comment:
@felixfbecker Alright, changed to `5.7.0`. I'll open a separate PR to update the types for `validator` `7.0.0`